spec/requests/api/v1/ecf_users_spec.rb

```diff
@@ -8,16 +8,27 @@
   let(:bearer_token) { "Bearer #{token}" }
 
   describe "#index" do
-    before :each do
-      # Heads up, for some reason the stored CIP IDs don't match
-      cip = create(:core_induction_programme, name: "Teach First")
-      school = create(:school)
-      cohort = Cohort.current
-      school_cohort = create(:school_cohort, school:)
-      mentor_profile = create(:mentor_participant_profile, school_cohort:, core_induction_programme: cip, cohort:)
-      create(:npq_participant_profile, school:)
-      create(:npq_participant_profile, school:, teacher_profile: mentor_profile.teacher_profile)
-      create_list(:ect_participant_profile, 2, school_cohort:, core_induction_programme: cip, cohort:)
+    # Heads up, for some reason the stored CIP IDs don't match
+    let(:core_induction_programme) { create(:core_induction_programme, name: "Teach First") }
+    let(:induction_programme) do
+      induction_programme = create(:induction_programme, :cip)
+      induction_programme.update!(core_induction_programme:)
+      induction_programme
+    end
+    let(:school) { create :school }
+    let(:cohort) { Cohort.current }
+    let(:school_cohort) { create :school_cohort, school: }
+
+    let(:mentor_profile) { create :mentor_participant_profile, school_cohort:, core_induction_programme:, cohort: }
+
+    let!(:npq_participant) { create :npq_participant_profile, school: }
+    let!(:npq_mentor_participant) { create :npq_participant_profile, school:, teacher_profile: mentor_profile.teacher_profile }
+    let!(:ecf_participants) { create_list :ect_participant_profile, 2, school_cohort:, core_induction_programme:, cohort: }
+
+    before do
+      Induction::Enrol.call(participant_profile: mentor_profile, induction_programme:, start_date: 2.months.ago)
+      Induction::Enrol.call(participant_profile: ecf_participants[0], induction_programme:, start_date: 2.months.ago)
+      Induction::Enrol.call(participant_profile: ecf_participants[1], induction_programme:, start_date: 2.months.ago)
```

Comment:
I think it'd be more readable to have `first_ecf_participant` and `second_ecf_participant` defined separately and referred to by their name rather than index.